core/build.gradle

```diff
@@ -106,6 +106,8 @@ if (isEclipse) {
       test.resources.srcDirs = ['resources']
     }
   }
+
+  testCompile 'org.elasticsearch:securemock:1.1'
```

Comment:
As it turns out I have this dependency in two places... here and in test-framework (which is what I intended):
https://github.com/elastic/elasticsearch/pull/13646/files#diff-830e6683f42cdce457750da0af4b29b6R35

I'll remove it here